### DISCUSSION TITLE
What is this part in Available plugins ?

### DISCUSSION THREAD

**Discussion by inson1** [pear-devs/pear-desktop#1438] [Q&A] [answered]
![brave_h6xCMua2ZU](https://github.com/th-ch/youtube-music/assets/75314629/2972d283-8685-49e1-89d9-5bea72abcdd1)

Its new plugin and most asked questions section (without any heading) ?
Yes, I dont see it in my version of the app. (that plugin)

**Comment by ArjixWasTaken** [comment, marked as answer]
If you are talking about the 'Auto confirm when paused' plugin, then it is not implemented as a plugin.

[Source code](https://github.com/th-ch/youtube-music/blob/06dc0e80f0611e47f780caa4869dd4d84b577ca4/src/renderer.ts#L116-L118) for reference.

**Reply by ArjixWasTaken** [reply]
And it has been like that for [2+ years](https://github.com/th-ch/youtube-music/commit/831b1ea8e12afde9bf31b2259efcc0eaeea30b45) already.

**Reply by ArjixWasTaken** [reply]
So I doubt any version you have doesn't have it.

**Reply by inson1** [reply]
@ArjixWasTaken You are right, its just feature :)
